tests/network/hotplug.go

```diff
@@ -44,91 +44,112 @@ import (
 	"kubevirt.io/kubevirt/tests/libnet"
 	"kubevirt.io/kubevirt/tests/libvmi"
 	"kubevirt.io/kubevirt/tests/libwait"
+	"kubevirt.io/kubevirt/tests/testsuite"
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-var _ = SIGDescribe("nic-hotplug", decorators.InPlaceHotplugNICs, func() {
+type hotplugMethod string
+
+const (
+	migrationBased hotplugMethod = "migrationBased"
+	inPlace        hotplugMethod = "inPlace"
+)
+const (
+	linuxBridgeName = "supadupabr"
+	ifaceName       = "iface1"
+	networkName     = "skynet"
+	vmIfaceName     = "eth1"
+)
+
+var hotPluggedVMI *v1.VirtualMachineInstance
+var hotPluggedVM *v1.VirtualMachine
```

Comment:
I think it's better to have the VM/VMI object closest to where it was used/created (instead of having it set as global to the whole suite) because it is easier to follow the test code also as time goes on and more tests are added.

It seems that this change was only for the `verifyHotplug()`, right?
If so, is it possible to change it to receive the vmi object and avoid using globals?

Reply:
+1

Reply:
done